pydoctor/epydoc/markup/restructuredtext.py

```diff
@@ -478,7 +477,80 @@ def visit_doctest_block(self, node: Node) -> None:
         else:
             self.body.append(flatten(colorize_doctest(pysrc)))
         raise SkipNode()
+    
+
+    # Other ressources on how to extend docutils: 
+    # https://docutils.sourceforge.io/docs/user/tools.html
+    # https://docutils.sourceforge.io/docs/dev/hacking.html
+    # https://docutils.sourceforge.io/docs/howto/rst-directives.html
+    # docutils apidocs: 
+    # http://code.nabla.net/doc/docutils/api/docutils.html#package-structure
+
+    # this part of the HTMLTranslator is based on sphinx's HTMLTranslator: 
+    # https://github.com/sphinx-doc/sphinx/blob/3.x/sphinx/writers/html.py#L271
+    def _visit_admonition(self, node: Node, name: str = '') -> None:
```

Comment:
While this is inspired by Sphinx's HTMLTranslator, we don't actually need to be compatible with that, so the default value for the `name` argument could be dropped. Then we also wouldn't need the `if` statement in the body, since nothing passes empty string for `name`.